FBSnapshotTestCase/FBSnapshotTestCase.modulemap

```diff
@@ -0,0 +1,6 @@
+framework module FBSnapshotTestCase {
+    umbrella header "FBSnapshotTestCase.h"
+
+    export *
+    module * { export * }
```

Comment:
Can you elaborate on this? or point me to docs?
How is that affecting the destination build? :mag: